### PR TITLE
fix: split layout of conflict view

### LINKED ISSUE #3950: Conflict resolution screen with different width
**Describe the bug**
It's hard to check if two files are the same if the conflict screen shows both at different width

**To Reproduce**
1. Create a conflict
2. Check the width

**Expected behavior**
The two versions should share the available screen space halve and halve.

**Screenshots**
grafik

**Client details:**
 - OS: ubuntu
 - Browser: ff 
 - Version: 26 rc 2
 - Device: desktop

Seen on cloud.nextcloud.com

### PR DESCRIPTION
**Pull request by max-nextcloud**
### 📝 Summary

Give both versions 50% of the screen space and try to vertically align them. This should make comparing the versions easier.

Fixes #3950.

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
Open test md in viewer -- displays conflicts | Open test md in viewer -- displays conflicts (2)

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [ ] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [x] Documentation is not required

### CODE REVIEW

File: cypress/e2e/sync.spec.js [hunk 1]
```diff
@@ -120,4 +118,32 @@ describe('Sync', () => {
 		cy.get('@close.all').should('have.length', 1)
 	})
 
+	it.only('reconnect does not lead to an empty file', () => {
```
Comment (mejo-) [inline cypress/e2e/sync.spec.js:121]: Is `it.only` wanted here? Seems like a developing leftover :thinking:

File: cypress/e2e/sync.spec.js
Comment (mejo-) [inline cypress/e2e/sync.spec.js:1]: Are changes to this file even meant to be part of the PR? They seem legit but unrelated :wink:

**Review by mejo- — CHANGES_REQUESTED** [review]

File: cypress/e2e/sync.spec.js
Comment (max-nextcloud) [reply cypress/e2e/sync.spec.js:1]: I was so sure i did not git commit -a ... but there it is.

**Review by mejo- — APPROVED** [review]

**Comment by mejo-** [comment]
/backport e83f2db0c35620b3174b929660d75686871c6f0d,21d48ec36dd49c63aaa7185e443b9d3043cffe60 to stable26

**Pull request merged**